openfl/component/aggregator/aggregator.py

```diff
@@ -55,6 +57,17 @@ def __init__(self,
                  log_metric_callback=None,
                  **kwargs):
         """Initialize."""
+        #INITIALIZE WANDB WITH CORRECT NAME
+        #The following variable, my_aggregator_name, can be changed to whatever you want. Right now I suppose that the name of the collaborators of the federation is in the format DATASETNAME_ENV_NUMBER
+        my_aggregator_name = '_'.join(set(element.split('_')[0] for element in authorized_cols))
```

Comment:
This should be moved into the `if LOG_WANDB` block because it's not used elsewhere. Also, it's a bit confusing that the `my_aggregator_name` variable is a combined list of collaborators. May want to modify this to:

```suggestion
        my_aggregator_name = 'Aggregator_' + '_'.join(set(element.split('_')[0] for element in authorized_cols))
```